src/visor/readable.go

```diff
@@ -145,10 +147,12 @@ func NewReadableTransactionOutput(t *coin.TransactionOutput, txid cipher.SHA256)
 }
 
 // NewReadableTransactionInput creates readable transaction input
-func NewReadableTransactionInput(uxID string, ownerAddress string) ReadableTransactionInput {
+func NewReadableTransactionInput(uxID, ownerAddress, coins, hours string) ReadableTransactionInput {
```

Comment:
Probably better to have `coins, hours uint64` and do the string conversions in here

Reply:
But then will have to do error handling for `coins` inside the function

Reply:
that's fine